ntbs-service/wwwroot/css/site.scss

```diff
@@ -147,3 +147,18 @@ body {
 .flex-container {
     display: flex;
 }
+
+.alerts-heading {
+    height: 100%;
+    margin: auto 0;
+}
+
+.tb-service-alerts-filter-label {
+    height: 100%;
+    margin: auto 0 auto auto;
+}
```

Comment:
I wonder if these are trying to build behaviour easier achieved with `flex`? Happy to be ignored here, just throwing out an idea :)